Source/SLKTextInputbar.h

```diff
@@ -26,7 +26,7 @@ typedef NS_ENUM(NSUInteger, SLKCounterPosition) {
 NS_ASSUME_NONNULL_BEGIN
 
 /** @name A custom tool bar encapsulating messaging controls. */
-@interface SLKTextInputbar : UIView
+@interface SLKTextInputbar : UIToolbar
```

Comment:
@dzenbot @hipwelljo regression?

Reply:
@dzenbot intentionally reverted that to UIToolbar

Reply:
I see, thanks